docs/configuration.md

```
@@ -237,13 +281,13 @@ Environment variables override config file settings but are overridden by CLI fl
 | Variable | Description | Example |
 |----------|-------------|---------|
 | `MURMUR_CLAUDE_PATH` | Path to claude executable | `/usr/local/bin/claude` |
-| `MURMUR_MODEL` | Model to use | `claude-sonnet-4-20250514` |
+| `MURMUR_MODEL` | Model to use | `claude-opus-4-5-20251101` |
 | `MURMUR_BACKEND` | Backend to use | `claude` or `cursor` |
 | `GITHUB_TOKEN` | GitHub Personal Access Token | `ghp_xxxx...` or `github_pat_xxxx...` |
 
 **Example:**
 ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)